ci/prow/make_config.go

```diff
@@ -0,0 +1,1010 @@
+// The make_config tool generates a full Prow config for the Knative project,
```

Comment:
does this file also need the `Copyright` stuff?

Reply:
Absolutely. Added.